src/lib/cli_commands/list_steps.rs

```diff
@@ -89,6 +90,14 @@ pub(crate) fn create_list(
 
             count = count + 1;
 
+            if &actual_task_name != key {
+                aliases
+                    .entry(actual_task_name)
+                    .or_default()
+                    .insert(key.clone());
+                continue;
+            }
```

Comment:
I put it here so that `count` doesn't change. But actually I don't know why `count` is needed. It seems it's only used for testing. 

I think a better approach for testing is simply assert the output string against a snapshot. And there's a good tool [expect_test](https://docs.rs/expect-test/latest/expect_test/) for such purpose, which supports auto-update the results. If you are interested, I can refactor the tests for you to take a look.

Reply:
Now I only manually tested with `cargo run --bin makers -- --list-all-steps`

![Uploading image.png…]()